web/packages/teleterm/src/ui/TopBar/Clusters/Clusters.tsx

```diff
@@ -53,17 +49,7 @@ export function Clusters() {
 
   function selectItem(clusterUri: ClusterUri): void {
     setIsPopoverOpened(false);
-    if (clusters.hasPendingAccessRequest) {
```

Comment:
It looks like the relevant field can be removed from `useClusters` as well, along with `clearPendingAccessRequest`.